iot_input_oca/controller/iot_input_controller.py

```diff
@@ -19,8 +19,7 @@ class CallIot(http.Controller):
     )
     def call_unauthorized_iot(self, serial, passphrase=False, *args, **kwargs):
         request = http.request
-        if not request.env:
-            return json.dumps(False)
+
```

Comment:
Why did you remove this?

Reply:
@etobella 
As I told you, this one has an issue with CodeCove. By the way, .env is part of the odoo rest controller so this is always true.

Reply:
Keep the original code. This might happen in some strange cases. Tests are unable to manage this kind of conditions. For that reason I was not expecting a 100% coverage